src/openzaak/conf/includes/api.py

```diff
@@ -25,6 +25,9 @@
 OPENZAAK_API_CONTACT_URL = "https://www.maykinmedia.nl"
 
 SPECTACULAR_SETTINGS = {
+    "SWAGGER_UI_DIST": "SIDECAR",
+    "SWAGGER_UI_FAVICON_HREF": "SIDECAR",
```

Comment:
I think these two settings are excessive since we don't provide swagger ui views, but I'll leave it up to you